src/clr-angular/forms/datepicker/date-container.spec.ts

```diff
@@ -94,6 +98,15 @@ export default function() {
 
                 sub.unsubscribe();
             });
+
+            // IE doesn't support MouseEvent constructors
+            itIgnore(["ie"], "marks the date control as touched when the datepicker popover is toggled", () => {
```

Comment:
This doesn't need to be wrapped so it skips IE.

Reply:
Yep. Initially I was create a `MouseEvent` here and thats why I added the `itIgnore`. But then later @youdz correctly pointed out that we didn't have to create the `MouseEvent` for this test. Will remove the `itIgnore`.